local_dev_files/destroy_dev_env.sh

```diff
@@ -75,7 +45,7 @@ for table in $dynamodb_tables; do
     aws dynamodb delete-table --table-name $table >/dev/null
 done
 
-printf "${greenColor}=> Destroying S3 terraform state${reset}\n"
+# printf "${greenColor}=> Destroying S3 terraform state${reset}\n"
```

Comment:
Any reason why this log has been disabled?

Reply:
It's an artifact of many commenting and uncommenting code.
Will fix.